portal-impl/src/portal.properties

```diff
@@ -6794,6 +6794,10 @@
     #
     feature.flag.LPS-134060.type=beta
 
+    feature.flag.LPS-183661.system=true
+
+    feature.flag.COMMERCE-9410.system=true
```

Comment:
Have you communicated these changes to the respective teams?

Reply:
No, I'll do that